tests/test_normalisation.py

```diff
@@ -16,15 +16,22 @@ def test_normalize_cupy():
     data = cp.asarray(host_data)
     flats = cp.asarray(host_flats)
     darks = cp.asarray(host_darks)
-    data = normalize_cupy(data, flats, darks)
 
     data_min = cp.array(-0.16163824, dtype=cp.float32)
     data_max = cp.array(2.7530956, dtype=cp.float32)
 
+    #--- testing normalize_cupy  ---#
+    data_normalize_cupy = normalize_cupy(data, flats, darks)
     for _ in range(10):
-        assert_allclose(cp.min(data), data_min, rtol=1e-05)
-        assert_allclose(cp.max(data), data_max, rtol=1e-05)
+        assert_allclose(cp.min(data_normalize_cupy), data_min, rtol=1e-05)
+        assert_allclose(cp.max(data_normalize_cupy), data_max, rtol=1e-05)
+    
+    #--- testing normalize_raw_cuda  ---#
+    data_normalize_raw_cuda = normalize_raw_cuda(data, flats, darks)
+    for _ in range(10):
+        assert_allclose(cp.min(data_normalize_raw_cuda), data_min, rtol=1e-05)
+        assert_allclose(cp.max(data_normalize_raw_cuda), data_max, rtol=1e-05)
```

Comment:
These tests fail locally. Looks like `normalize_cupy` and `normalize_raw_cuda` are not giving the same results.
```suggestion
    data_normalize_raw_cuda = normalize_raw_cuda(data, flats, darks)
    data_max = cp.array(2.3314207, dtype=cp.float32)
    for _ in range(10):
        assert_allclose(cp.min(data_normalize_raw_cuda), data_min, rtol=1e-05)
        assert_allclose(cp.max(data_normalize_raw_cuda), data_max, rtol=1e-05)
```